samples/SkiaMultiplatformSample/build.gradle.kts

```diff
@@ -165,11 +177,23 @@ kotlin {
             val macosArm64Main by getting {
                 dependsOn(macosMain)
             }
-            val iosMain by getting {
+            val uikitMain by creating {
                 dependsOn(darwinMain)
             }
+            val iosMain by getting {
+                dependsOn(uikitMain)
+            }
             val iosSimulatorArm64Main by getting {
-                dependsOn(iosMain)
+                dependsOn(uikitMain)
```

Comment:
Why not `dependsOn(iosMain)`?

Reply:
Because there are some different actuals for iOS and tvOS for clipboard and multitouch, but otherwise the code can be shared.

Reply:
My bad, will fix

Reply:
Fixed, forcepushed